docs/build/step5.md

```diff
@@ -56,49 +57,51 @@ There used to be just one option. Now there are more, so you have to make a deci
 
-What influences this distance? The biggest influences are (1) body-blocking and (2) "noisy" environments. The human body is a lot of water, and water is an excellent blocker of wireless communication. So, sleeping on a pod and smothering it entirely with your body will possibly decrease the ability of radio-link to communicate with the pod. Environments with a high concentration of wireless signals can also interfere with radio-link communications and make closer proximity a benefit. Where might those kinds of situations happen? Concerts, conferences, and sporting arenas are pretty prone to interference.
+Many people keep their device on the same side of their body as their pump during the day. They use a pocket, carabiner, lanyard, SPIbelt - the options are endless. What you don't want to do is put it in a blocking bag that has RFID blocking (some travel fanny packs have that).  
 
-Many people keep their radio-link on the same side of their body as their pump during the day. They use a pocket, carabiner, lanyard, SPIbelt - the options are endless. What you don't want to do is put the radio-link in a blocking bag that has RFID blocking (some travel fanny packs have that).  
+The practical range can vary by device design and among individual units of the same design. Do some experiments - send a communication signal from phone to pump - if it makes the round-trip, extend the distance and repeat the test.  Figure out what works for your devices.
 
-The practical range can vary from type to type and among individual units. Do some experiments - send a communication signal from phone to pump - if it makes the round-trip, extend the distance and repeat the test.  Figure out what works for your devices.
+Suggested communication signals to send:
 
-Suggested communication signal to send for **Medtronic**: Button Press
+* **Medtronic**: Button Press
+* **Omnipod**: Read Pod Status
 
-Suggested communication signal to send for **Omnipod**: Read Pod Status
+If you use Omnipod and have a newer version of Loop, the report from Read Pod status has the Received Signal Strength Indicator (RSSI) between the RileyLink compatible device and the Pod
 
-* If you have a newer version of Loop, the report from Read Pod status has the Received Signal Strength Indicator (RSSI) between the radio-link and the Pod
-* The RSSI is a positive number with a larger number indicating a stronger signal strength detected by the Pod from the radio-link
+* The RSSI is a positive number with a larger number indicating a stronger signal strength detected by the Pod
```

Comment:
Even though the official Insulet documentation standard is to capitalize "Pod", it would be better to use "pod" instead to match the rest of loopdocs which didn't adopt this Insulet documentation standard.